formulas/functions/statistical.js

```diff
@@ -251,8 +251,30 @@ const StatisticalFunctions = {
 
   MIN: () => {},
 
-  MINA: () => {},
-
+  MINA: (...values) => {
+    const arr = values.map(a => a.value)
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(isNaN(a - a))
+        return 0
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      if(["boolean", "string"].includes(typeof a))
+        return 0
+      if(typeof a === "number")
+        return a
+      return null
+    })
+    const finalArr = updatedArr.filter((value) => value != null)
+    if(finalArr.length === 0)
+      return 0;
+    //used instead of Math.max(...finalArr) to avoid JS maximum input length
+    return finalArr.reduce((a, b) => Math.min(a, b), Infinity)
```

Comment:
It is arguably more readable the way you have it (separate `map()` and `reduce()`), but it's definitely possible to do all the work inside of `reduce()`, (can even do the equivalent of `filter` inside the `reduce`, though again, I don't think it's necessary).

Instead of parsing input values in a `map()` loop (loop 1), then using `reduce()` to compare each one in order to find the min/max (loop2), parse values and then compare,  all inside of `reduce()`

Reply:
E.G:

```js
const parseNumA = (a) => {
  if (typeof a === "boolean")
    return +a;
  if (typeof a === "string")
    return 0;
  if (isNaN(a - a)) // handle Infinity/-Infinity/NaN
    return 0;
  if (typeof a === "number")
    return a;
  return null;
}

const min = arr.reduce((min, val) => {    
  const num = parseNumA(val);
  if (num !== null && num < min)
    return num;
  return min;
}, Infinity);

if (min === Infinity) // all given values were null
  return 0;

return min;
```

(note: with a check for `null` in the reducer plus a check for `Infinity` after the reducer, this also avoids the need to Array.filter)

Reply:
Hmmm this is really cool! Ill modify it for the next implementation